content/src/fundings/food-agriculture-grant.md

```diff
@@ -1,5 +1,5 @@
 ---
-filename: Food and Agriculture Research & Development Grant Program
+filename: food-agriculture-grant
```

Comment:
I've seen a couple instances where we've gone from a kebab-case to title case and vice versa. Are these changes you're making or did this happen when you pulled in content? Just wondering if there's a rhyme or reason here.

Reply:
This fix is a result of campfire rules. If there's spaces in the filename, it creates the filename with underscores, which is not the convention in the rest of the project.

Reply:
Definietly should make it write-once-read-only, and maybe we should create a variant that requires nospaces as well.